packages/shared/styles/components/organisms/SfCarousel.scss

```diff
@@ -37,4 +38,8 @@
     --carousel-controls-size: 15rem;
     --carousel-controls-display: flex;
   }
+  @supports (-webkit-touch-callout: none) {
+    --carousel-width: 100vw;
```

Comment:
@AdamPawlinski is a good idea? When you set some `max-width:` for container and use inside it carousel than the carousel want to be have 100vw ;)

Reply:
It was only solution that works for me. If I use px it would be nice on some screens and bad on others. Using 100% also doesn't work because it shows only every 4th item of carousel (strange).